pkg/k8sutil/merge_test.go

```diff
@@ -154,8 +153,6 @@ func TestMergePatchContainersOrderPreserved(t *testing.T) {
 				build("c6", "image:E"),
 			},
 		)
-		if diff != "" {
-			t.Fatalf("patch result did not match. diff:\n%s", diff)
-		}
+		require.Empty(t, diff, "patch result did not match. diff:\n%s", diff)
```

Comment:
> Is this the same as an "equal" comparison? 🤔
> 
> If it is, what do you think about just using a require.Equal instead of asserting the diff is empty?

I will replace it here also